src/Defra.PTS.Web.UI/Views/Shared/_FooterPartial.cshtml

```diff
@@ -1,4 +1,11 @@
-﻿@using Defra.PTS.Web.UI.Constants;
+﻿@using Defra.PTS.Web.UI.Constants
+@inject Microsoft.AspNetCore.Http.IHttpContextAccessor context
+
+@{
+    var locale = context.HttpContext.Request.Cookies["locale"] ?? "en";
```

Comment:
It's had to change to en-GB for GB sorry

Reply:
completed